lib/config/presets/internal/replacements.ts

```diff
@@ -91,6 +93,32 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'react-query-devtools-to-scoped': {
+    description:
+      '`react-query/devtools` became scoped under the `tanstack` organization.',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=2.6.3 <4.0.0',
```

Comment:
Do we want people to be on the latest `non-scoped` version before we offer the replacement PR. Or do we just want them to be on any `2.x.x` version?

Reply:
Looking at the other replacements, it appears that we've preferred any version of the last major version. If I recall correctly the reasoning is:
1. We minimize the number of PRs by upgrading them early 	
3. Any upgrade is going to be a breaking change regardless of which minor or patch version they were on before

I think the guidelines should be:
1. `matchCurrentVersion` should be a range of the last major version of the old package
2. `replacementVersion` should be a fixed version with the first version of the new package

What do you think?

Reply:
In some edge cases we don't want them to be *downgraded* though if the library published in parallel for some time.

Reply:
```suggestion
        matchCurrentVersion: '>=2.0.0 <4.0.0',
```

Sound like we want this then?

Reply:
> In some edge cases we don't want them to be downgraded though if the library published in parallel for some time.

@rarkins can you explain some more?

Reply:
Ignore what I wrote